### PR TITLE
add sea61 m50

### PR DESCRIPTION
**Pull request by callumrollo**

### CODE REVIEW

**Review by akmazur — COMMENTED** [review]
Typos

File: mission_yaml/SEA61_M50.yml [hunk 1]
```diff
@@ -0,0 +1,459 @@
...
+  project_url: https://voiceoftheocean.org/samba-smart-autonomous-monitoring-of-the-baltic-sea/
+  summary: ' Part of SAMBA continuous monitoring'
+  # Examples: Skagerrak, Kattegatt, Baltic, Gulf of Bothnia, Gulf of Finland, Gulf of Riga
+  sea_name: Baltic
+
+glider_devices:
+  altimeter:
+    make: UNKNOWN
+    model: UNKNOWN
+    serial: '048'
```
Comment (akmazur) [inline mission_yaml/SEA61_M50.yml:52]: serial: '48'

File: mission_yaml/SEA61_M50.yml [hunk 2]
```diff
@@ -0,0 +1,459 @@
...
+    calibration_date: " 2021-03-23"
+    calibration_parameters: {Chl_DarkCounts: 46, Chl_SF: 0.0073, PC_DarkCounts: 42, PC_SF: 0.0422, SIG_DarkCounts: 45, SIG_SF: 1.798E-06}
+  oxygen:
+    make: JFE Advantech
+    model: AROD-FT
+    serial: '0038'
+    long_name: JFE Advantech RINKO-FT
+    make_model: JFE Advantech AROD_FT
+    factory_calibrated: "Yes"
+    calibration_date: "2021-01-13"
```
Comment (akmazur) [inline mission_yaml/SEA61_M50.yml:77]: 2020-02-05

File: mission_yaml/SEA61_M50.yml [hunk 3]
```diff
@@ -0,0 +1,459 @@
...
+  irradiance:
+    make: SeaBird
+    model: OCR504
+    serial: '2193'
+    long_name: SeaBird OCR504
+    make_model: SeaBird OCR504
+    factory_calibrated: "Yes"
+    calibration_date: "2020-02-02"
```
Comment (akmazur) [inline mission_yaml/SEA61_M50.yml:85]: 2021-02-02

File: mission_yaml/SEA61_M50.yml [hunk 4]
```diff
@@ -0,0 +1,459 @@
...
+    standard_name: raw_chlorophyll_in_sea_water
+    units:        counts
+    coordinates:  time depth latitude longitude
+    instrument:    instrument_flourometer
+
+  phycocyanin:
+    source:       FLBBPC_PC_SCALED
+    long_name:    phycocyanin
+    standard_name: concentration_of_phycocyanin_in_sea_water
+    units:        ppb
```
Comment (akmazur) [inline mission_yaml/SEA61_M50.yml:245]: mg m-3

File: mission_yaml/SEA61_M50.yml [hunk 5]
```diff
@@ -0,0 +1,459 @@
...
+    standard_name: raw_phycocyanin_in_sea_water
+    units:        counts
+    coordinates:  time depth latitude longitude
+    instrument:    instrument_flourometer
+
+  backscatter:
+    source:       FLBBPC_BB_700_SCALED
+    long_name:    700 nm wavelength backscatter
+    standard_name: 700_nm_wavelength_backscatter_in_sea_water
+    units:        m-1
```
Comment (akmazur) [inline mission_yaml/SEA61_M50.yml:265]: m-1 sr-1

File: mission_yaml/SEA61_M50.yml [hunk 6]
```diff
@@ -0,0 +1,459 @@
...
+    coordinates:  time depth latitude longitude
+    instrument:    instrument_scatterometer
+
+
+
+  oxygen_concentration:
+    source:       AROD_FT_DO
+    long_name:    oxygen concentration
+    standard_name: mole_concentration_of_dissolved_molecular_oxygen_in_sea_water
+    units:        umol l-1
```
Comment (akmazur) [inline mission_yaml/SEA61_M50.yml:287]: mmol m-3

**Review by akmazur — COMMENTED** [review]
typos and corrected dates

**Comment by callumrollo** [comment]
Thanks for the catches!

**Pull request merged**